### PR TITLE
Don't build images, to make the getting started guides run faster

### PR DESCRIPTION
**Pull request by pchico83**
Now our samples are executed in SH, users cannot take advantage of smart builds when running our samples.
This PR updates our samples to use pre-built images and speed up the execution of the sample.

This PR must be merged in combination with:
- https://github.com/okteto/movies-with-helm/pull/9
- https://github.com/okteto/movies-with-compose/pull/15
- https://github.com/okteto/aspnetcore-getting-started/pull/12
- https://github.com/okteto/go-getting-started/pull/18
- https://github.com/okteto/java-gradle-getting-started/pull/6
- https://github.com/okteto/java-maven-getting-started/pull/9
- https://github.com/okteto/node-getting-started/pull/10
- https://github.com/okteto/php-getting-started/pull/11
- https://github.com/okteto/python-getting-started/pull/16
- https://github.com/okteto/ruby-getting-started/pull/11

### CODE REVIEW

**Review by rlamana — APPROVED** [review]

**Review by ifbyol — APPROVED** [review]

**Comment by rberrelleza** [comment]
Building is something that every user will do. I'm concerned that removing this from the tutorials will make it harder for users to understand how to use Okteto.

Do we have data that can help us understand if having slower samples is causing us to lose evaluations, trials, or POCs?  Or what is the motivation behind investing on this change in the user experience?

File: src/content/guides/samples/aspnetcore.mdx
Comment (rberrelleza) [inline src/content/guides/samples/aspnetcore.mdx:1]: The samples under guides/samples are supposed to be the 'reference' example of how to develop with each programming language. I don't think it's a good idea to remove the build step from here.

Comment (rlamana) [reply src/content/guides/samples/aspnetcore.mdx:1]: The content of the guide is the same, take the user to execute the "deploy" command. Removing the "--build" parameter simplifies it, as well as accelerates the execution for a better first impression. We can always add an explanatory paragraph about the deploy and how it implies to build images (an mention the parameter to force the build if you like), but I honestly think this helps a lot to follow the sample.

Comment (rberrelleza) [reply src/content/guides/samples/aspnetcore.mdx:1]: Thanks for the explanation!

Comment (pchico83) [reply src/content/guides/samples/aspnetcore.mdx:1]: @rberrelleza we are adding a dedicated doc explaining how to configure the "build" section. We won't merge this PR until that guide is out

Comment (rberrelleza) [reply src/content/guides/samples/aspnetcore.mdx:1]: Cool, thanks for explaining @pchico83 !

**Comment by pchico83** [comment]
Merging these changes now we have an open PR for onboarding your own app to Okteto:
https://github.com/okteto/docs/pull/609

**Pull request merged**